test/identityMetadata.test.ts

```diff
@@ -0,0 +1,67 @@
+import { Random, RandomCharacterRange } from "@nmshd/transport";
+import { ConnectorClientWithMetadata, Launcher } from "./lib/Launcher";
+import { ValidationSchema } from "./lib/validation";
+
+const launcher = new Launcher();
+let client1: ConnectorClientWithMetadata;
+
+beforeAll(async () => {
+    [client1] = await launcher.launch(1);
+}, 30000);
+
+afterAll(() => launcher.stop());
+
+describe("IdentityMetadata", () => {
+    test.each([
+        {
+            reference: "did:e:localhost:dids:1234567890abcdef123456",
+            key: undefined,
+            value: "value"
+        },
+        {
+            reference: "did:e:localhost:dids:1234567890abcdef123456",
+            key: undefined,
+            value: { a: "json" }
+        },
+        {
+            reference: "did:e:localhost:dids:1234567890abcdef123456",
+            key: "key",
+            value: "value"
+        }
+    ])("should upsert an IdentityMetadata with key '$key' and value '$value'", async (data) => {
```

Comment:
If the `key` is undefined, the test name isn't optimal, as it looks like the string `'undefined'` is used as `key`.

Reply:
yep, we have the same problem in the runtime, I see no possibility to make this safe :/

Reply:
Yes that's right, not easy to refactor without increasing the number of lines of code excessively. 😞 Still hoping for a better tomorrow 🙏